src/morse-code.js

```diff
@@ -8,6 +9,90 @@
  * @returns {string}  - String converted to english or morse code
 */
 
-function morseCode(message, convertTo) { }
+function morseCode(message, convertTo) {
+  let morse = {
+    'A': '.-',
+    'B': '-...',
+    'C': '-.-.',
+    'D': '-..',
+    'E': '.',
+    'F': '..-.',
+    'G': '--.',
+    'H': '....',
+    'I': '..',
+    'J': '.---',
+    'K': '-.-',
+    'L': '.-..',
+    'M': '--',
+    'N': '-.',
+    'O': '---',
+    'P': '.--.',
+    'Q': '--.-',
+    'R': '.-.',
+    'S': '...',
+    'T': '-',
+    'U': '..-',
+    'V': '...-',
+    'W': '.--',
+    'X': '-..-',
+    'Y': '-.--',
+    'Z': '--..',
+    '0': '-----',
+    '1': '.----',
+    '2': '..---',
+    '3': '...--',
+    '4': '....-',
+    '5': '.....',
+    '6': '-....',
+    '7': '--...',
+    '8': '---..',
+    '9': '----.',
+    ',': '--..--',
+    '?': '..--..',
+    '@': '.--.-.',
+    '+': '.-.-.',
+    '-': '-....-',
+    '/': '-..-.',
+    ' ': '  '
+  }
+
+  let translatedToMorse = '';
+  let translatedToEnglish = '';
+  let messageTranslated = '';
+
+  if (convertTo === 'morse') {
+    for (let i = 0; i < message.length; i++) {
+
+      let morseValue = morse[`${message[i].toUpperCase()}`];
+
+      if (morseValue != undefined) {
+        translatedToMorse += `${morseValue}`;
+
+        if (message[i] !== ' ' && i !== message.length - 1) {
+          translatedToMorse += ' ';
+        }
+      }
+    }
+
+    messageTranslated = translatedToMorse;
+  }
+
+  if (convertTo === 'english') {
```

Comment:
Since the two possible options of `converTo `are `english `and `morse`. You could have used `if () {} else {}` instead of:

````
if(converTo === 'morse') {
  ...
}

if(converTo === 'english') {
  ...
}
````